share/templates/08-Orchestral/03-String_Orchestra.mscx

```diff
@@ -52,6 +52,7 @@
         </section>
       <section id="brass" brackets="true" showSystemMarkings="false" barLineSpan="true" thinBrackets="true">
         <family>horns</family>
+        <family>flugelhorns</family>
```

Comment:
Same here, update the <code>orchestral</code> score order in <code>orders.xml</code>.